unblob/extractor.py

```diff
@@ -15,6 +17,14 @@
 FILE_PERMISSION_MASK = 0o644
 DIR_PERMISSION_MASK = 0o775
 
+# re-exported, or unused symbols
+__all__ = [
+    "is_safe_path",
+    "MaliciousSymlinkRemoved",
+    "fix_symlink",
+    "is_recursive_link",
+]
+
```

Comment:
Why is this required ?

Reply:
answered as main comment.
these changes are to be dropped.